src/NDItk/multigroup/Metadata.hpp

```diff
@@ -29,9 +29,11 @@ class Metadata {
   base::SingleRealRecord atomic_weight_;
   base::SingleRealRecord temperature_;
   base::SingleRealRecord dilution_;
-  base::SingleIntegerRecord reactions_;
   base::SingleIntegerRecord primary_groups_;
   std::vector< base::SingleIntegerRecord > outgoing_groups_;
+  base::SingleIntegerRecord upscatter_groups_;
+  base::SingleIntegerRecord downscatter_groups_;
+  base::SingleIntegerRecord reactions_;
```

Comment:
Is this the number of reactions? If so, adding number or something would make it clearer that it's not the actual reaction XSs. Even though SingleIntegerRecord implies it's one int...

Reply:
yes, the number of reactions

Reply:
I'll change that

Reply:
changed in fix/review-part3